mkw_stats_bot/mkw_stats/handlers/message_manager.py

```
@@ -0,0 +1,67 @@
+        self,
+        message: discord.Message,
+        embed: discord.Embed,
+        countdown_seconds: int = 30,
+    ):
+        """Show countdown timer and delete message (for regular discord messages)."""
+        for remaining in range(countdown_seconds, 0, -1):
+            await asyncio.sleep(1)
+            if remaining <= 5:
+                try:
+                    await message.edit(embed=embed, content=f"Disappearing in {remaining} seconds...")
+                except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)